files/.template-lintrc.cjs

```diff
@@ -2,4 +2,5 @@
 
 module.exports = {
   extends: 'recommended',
+  checkHbsTemplateLiterals: false,
```

Comment:
this disables parsing of js and ts files (which, for components (where we'd want to use template-lint), we don't support in the v2 addon blueprint)